hack/testing/test-upgrade.sh

```diff
@@ -32,6 +32,32 @@ function dumpEvents() {
 }
 trap dumpEvents EXIT
 
+function createOldIndexPattern() {
+  echo "creating index with old pattern"
+
+  indexDate=`date +%Y.%m.%d`
+  #rawUUID="$(tr -dc 'a-f0-9' < /dev/urandom | head -c 32)"
+
+  #echo $rawUUID
+  #echo ${rawUUID:0:8}-${rawUUID:8:4}-${rawUUID:12:4}-${rawUUID:16:4}-${rawUUID:20}
+
+  #genUUID="$(echo ${rawUUID:0:8}-${rawUUID:8:4}-${rawUUID:12:4}-${rawUUID:16:4}-${rawUUID:20})"
+  genUUID="7601c01e-012e-7640-42ab-01e833d4fd33"
+
+  esPod=`oc get pods -l component=es -o name | sed "s,pod/,,"`
+
+  waitFor "[[ \"Running\" == \"\$(oc get pods -l component=es -o jsonpath='{.items[*].status.phase}')\" ]]" "$(( 3 * TIME_MIN ))" || return 1
+  [ -z "$esPod" ] && echo "Unable to find ES pod for recreating old index pattern" && return 1
+
+  # create an old index pattern
+  oc exec $esPod -- curl --cacert /etc/elasticsearch/secret/admin-ca --cert /etc/elasticsearch/secret/admin-cert --key /etc/elasticsearch/secret/admin-key \
+                         -XPUT "https://logging-es:9200/oldindex.${genUUID}.${indexDate}" -d '{ "settings": { "index": { "number_of_shards": 1, "number_of_replicas": 0 } } }'
+
+  # necessary?
+  oc exec $esPod -- curl --cacert /etc/elasticsearch/secret/admin-ca --cert /etc/elasticsearch/secret/admin-cert --key /etc/elasticsearch/secret/admin-key \
+                         -XGET "https://logging-es:9200/_cluster/health/oldindex.${genUUID}.${indexDate}?wait_for_status=yellow&timeout=50s"
```

Comment:
yeah, i'll remove it; i just wanted to make sure that the index was available first before we check for it in the upgrade. doesn't seem to be necessary